src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -660,6 +690,23 @@ def _get_clone_source(self):
             raise VolumeException(-errno.EINVAL, "error fetching subvolume metadata")
         return clone_source
 
+    def _get_clone_progress(self):
+        try:
+            src_snappath = self.metadata_mgr.get_option("source", "snappath")
+            src_usedbytes = int(self.fs.getxattr(src_snappath, 'ceph.dir.rbytes'))
+            clone_usedbytes = int(self._recursive_size(self.path))
+        except MetadataMgrException as me:
+            raise VolumeException(-errno.EINVAL, "error fetching clone progress metadata - {0}".format(me))
+        except cephfs.Error as e:
+            raise VolumeException(-e.args[0], "error fetching clone progress metadata - {0}".format(e.args[1]))
+
+        progress = "0.00" if src_usedbytes == 0 else '{0:.2f}'.format((clone_usedbytes / src_usedbytes) * 100.0)
```

Comment:
if possible, please use `f""` format strings ... they are more intuitive to understand